packages/react-paypal-js/src/v6/hooks/usePayLaterOneTimePaymentSession.ts

```diff
@@ -61,6 +61,19 @@ export function usePayLaterOneTimePaymentSession({
         });
         sessionRef.current = newSession;
 
+        const handleReturnFromPayPal = async () => {
+            const isResumeFlow = newSession.hasReturned?.();
+            if (isResumeFlow) {
+                try {
+                    await newSession.resume?.();
+                } catch (err) {
+                    setError(err as Error);
+                }
+            }
+        };
+
+        handleReturnFromPayPal();
```

Comment:
Should we only do this when the presentation mode is redirect/direct app switch? Seems like an unnecessary code path for all other presentation modes 🤔